lib/store/oracle.ex

```diff
@@ -33,7 +33,7 @@ defmodule Elixium.Store.Oracle do
   """
   @spec inquire(pid, tuple) :: any()
   def inquire(pid, options) do
-    GenServer.call(pid, options)
+    GenServer.call(pid, options, 20000)
```

Comment:
Why are we adding a timeout? It defaults to infinity without a timeout

Reply:
Had a few weird issues during the tests & in wallet with the inquire needing the timeout, the genserver call defaults to a timeout of 5000ms without it